### PR TITLE
hyperv: Powershell was not found in the path 

### PR DESCRIPTION
**Pull request by RA489**
hyperv: Powershell was not found in the path 
fixes https://github.com/kubernetes/minikube/issues/4606

### CODE REVIEW

File: pkg/minikube/problem/err_map.go [hunk 1]
```diff
@@ -144,6 +144,12 @@ var vmProblems = map[string]match{
 		Advice: "Configure an external network switch following the official documentation, then add `--hyperv-virtual-switch=<switch-name>` to `minikube start`",
 		URL:    "https://docs.docker.com/machine/drivers/hyper-v/",
 	},
+	"HYPERV_POWERSHELL_NOT_FOUND": {
+		Regexp: re(`Powershell is not found in the path`),
```
Comment (medyagh) [inline pkg/minikube/problem/err_map.go]: Thanks for making this PR, please make sure the regex, matches the exact error message,
in this case it is "Powershell was not found in the path"

Comment (RA489) [reply pkg/minikube/problem/err_map.go]: s/is/was

**Comment by RA489** [comment]
@medyagh Thanks for your reviews! I have updated as per your comments.

**Comment by blueelvis** [comment]
@medyagh @RA489 - This same set of functionality is available with Docker machine's Hyper-V implementation but these are not public :(

https://github.com/machine-drivers/machine/blob/master/drivers/hyperv/powershell.go

**Pull request merged**